recipients_app/.run/prod.run.xml

```diff
@@ -1,7 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Do you delete this on purpose?

Reply:
Hmm, I think I removed some run configurations in WebStorm that were suddenly there. I've never shared IDE run configurations in a repository and I don't think we should. What do you think?